docs/apache-airflow/best-practices.rst

```diff
@@ -577,3 +577,41 @@ For connection, use :envvar:`AIRFLOW_CONN_{CONN_ID}`.
     conn_uri = conn.get_uri()
     with mock.patch.dict("os.environ", AIRFLOW_CONN_MY_CONN=conn_uri):
         assert "cat" == Connection.get("my_conn").login
+
+Metadata DB maintenance
+-----------------------
+
+Over time, the metadata database will increase its storage footprint as more dag and task runs and event logs accumulate.
+
+You can use the Airflow CLI to purge old data with command ``airflow db clean``.
+
+See :ref:`db clean usage<cli-db-clean>` for more details.
+
+Upgrades and downgrades
+-----------------------
+
+Backup your database
+^^^^^^^^^^^^^^^^^^^^
+
+It's always a wise idea to backup the metadata database before undertaking any operation modifying the database.
+
+Disabling the scheduler
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You might consider disabling the Airflow cluster while you perform such maintenance.  One way to do so would be to set the param ``[scheduler] > use_job_schedule`` and wait for any running dags to complete; after this no new dag runs will be created unless externally triggered.
+
+Another way to accomplish roughly the same thing is to use the ``dags pause`` command.  You *must* keep track of the dags that are paused before you begin this operation, otherwise when it comes time to unpause, you won't know which ones should remain paused!  So first run ``airflow dags list``, then store the list of unpaused dags, and keep this list somewhere so that later you can unpause only these.
```

Comment:
```suggestion
Another way to accomplish roughly the same thing is to use the ``dags pause`` command.  You *must* keep track of the DAGs that are paused before you begin this operation, otherwise when it comes time to unpause, you won't know which ones should remain paused!  So first run ``airflow dags list``, then store the list of unpaused DAGs, and keep this list somewhere so that later you can unpause only these.
```